doc/releases/release-notes-4.1.rst

```diff
@@ -92,6 +92,10 @@ Drivers and Sensors
 
 * Display
 
+  * Added API function :c:func:`display_show` and display capability :c:enum:`SCREEN_INFO_REQUIRES_SHOW`
+    for displays that allow tearing prevention of segmented updates. (:github:`79936`)
+  * Implemented :c:func:`display_show` for SDL display driver (:dtcompatible:`zephyr,sdl-dc`) and added display capability :c:enum:`SCREEN_INFO_REQUIRES_SHOW`. (:github:`79936`)
```

Comment:
Nit: try to keep the line length <= 100 chars.

Reply:
Fixed